docs/src/content/docs/fr/guides/customization.mdx

```diff
@@ -344,23 +345,39 @@ Pour utiliser Google Fonts, suivez le [Guide de configuration de Fontsource](#co
 Le projet [Fontsource](https://fontsource.org/) simplifie l’utilisation des polices Google et d’autres polices open source.
 Il fournit des modules npm que vous pouvez installer pour les polices que vous souhaitez utiliser et inclut des fichiers CSS prêts à l’emploi à ajouter à votre projet.
 
-1. Trouvez la police que vous souhaitez utiliser dans le [catalogue de Fontsource](https://fontsource.org/). Cet exemple utilisera [IBM Plex Serif](https://fontsource.org/fonts/ibm-plex-serif).
+1.  Trouvez la police que vous souhaitez utiliser dans le [catalogue de Fontsource](https://fontsource.org/). Cet exemple utilisera [IBM Plex Serif](https://fontsource.org/fonts/ibm-plex-serif).
 
-2. Installez le package pour la police choisie. Vous pouvez trouver le nom du package en cliquant sur "Installer" sur la page de police Fontsource.
+2.  Installez le package pour la police choisie. Vous pouvez trouver le nom du package en cliquant sur "Installer" sur la page de police Fontsource.
 
-<Tabs>
-	<TabItem label="npm">
-		```sh npm install @fontsource/ibm-plex-serif ```
-	</TabItem>
-	<TabItem label="pnpm">
-		```sh pnpm install @fontsource/ibm-plex-serif ```
-	</TabItem>
-	<TabItem label="Yarn">```sh yarn add @fontsource/ibm-plex-serif ```</TabItem>
-</Tabs>
+        <Tabs>
+
+    <TabItem label="npm">
+
+            ```sh
+            npm install @fontsource/ibm-plex-serif
+            ```
+
+          </TabItem>
+          <TabItem label="pnpm">
+
+            ```sh
+            pnpm install @fontsource/ibm-plex-serif
+            ```
+
+          </TabItem>
+          <TabItem label="Yarn">
+
+            ```sh
+            yarn add @fontsource/ibm-plex-serif
+            ```
+
+        </TabItem>
+
+      </Tabs>
```

Comment:
This is not part of #1102 but when editing the portion below, I realized that this one broken for a while so I fixed it.

Reply:
Ah yeah… that’s an unfortunate Prettier issue where it collapses the code block. Thanks for fixing!